keyboards/lily58/keymaps/perugi/keymap.c

```diff
@@ -77,39 +77,39 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * `-----------------------------------------/       /     \      \-----------------------------------------'
- *                   | LAlt | LGUI |LOWER | /Space  /       \Enter \  |RAISE |BackSP| SLO  |
+ *                   | LAlt | LGUI |LOWER | /Space  /       \Backsp\  |RAISE |BackSP| SLO  |
  *                   |      |      |      |/       /         \      \ |      |      |      |
  *                   `----------------------------'           '------''--------------------'
  */
 [_LOWER] = LAYOUT(
-  KC_GRV,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
-  KC_F1,      KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                           KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
-  KC_MINS,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_EQL,
+  KC_GRV,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+  KC_F1,      KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+  KC_MINS,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_EQL,
   S(KC_MINS), S(KC_1), S(KC_2), S(KC_3), S(KC_4), S(KC_5), KC_LCBR, KC_RCBR, S(KC_6), S(KC_7), S(KC_8), S(KC_9), S(KC_0), S(KC_EQL),
-                                _______, _______, _______, _______,    _______,    _______, _______, _______
+                                _______, _______, _______, _______, KC_BSPC, _______, _______, _______
 ),
 
 /* RAISE
- * System command layer with VIM-inspired assignments
+ * System command layer with VIM-inspired assignments and alternate locations for some hard-to-reach keys.
  * ,-----------------------------------------.                    ,-----------------------------------------.
  * |      |      |      |      |      |      |                    |      |      |      |      |      |      |
  * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
- * |      |      |      | END  | Redo |      |                    |      | Undo |      |      |PrntSc|      |
+ * |   `  |      |      | END  | Redo |      |                    |      | Undo |      |      |PrntSc|      |
  * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
- * |      |      |      | PGDN | PGUP | H/E  |-------.    ,-------| Left | Down |  Up  |Right |      |      |
+ * |   _  |   !  |      | PGDN | PGUP | H/E  |-------.    ,-------| Left | Down |  Up  |Right |      |      |
  * |------+------+------+------+------+------|   {   |    |    }  |------+------+------+------+------+------|
  * |      |      |  DEL |      |      | BEG  |-------|    |-------|      |      |      |      |Search|      |
```

Comment:
Delete cannot be accessed with only one hand, so it cannot be pressed while using a mouse. The same can be said about ordinary keebs, but it would be interesting to try.

Reply:
You're right, also the alternate locations were not very comfortable, so I reverted to original only. I suggest that we think about this more and create another PR in the future if we come up with something else.